resources/public/api/conf/3.0/employment_expenses.yaml

```diff
@@ -0,0 +1,22 @@
+put:
+  $ref: "./employment_expenses_create_and_amend.yaml"  
+  summary: Create and Amend Employment Expenses{{#maybeTestOnly "3.0 employment-expenses-create-and-amend"}}{{/maybeTestOnly}}
```

Comment:
The flags I commented on, are still here and in V2

Reply:
ah sorry misinterpreted